src/Illuminate/Cache/Repository.php

```diff
@@ -554,7 +554,8 @@ protected function getMinutes($duration)
         $duration = $this->parseDateInterval($duration);
 
         if ($duration instanceof DateTimeInterface) {
-            $duration = Carbon::now()->diffInSeconds(Carbon::createFromTimestamp($duration->getTimestamp()), false) / 60;
+            $dateFactory = app(Factory::class);
+            $duration = $dateFactory->now()->diffInSeconds($dateFactory->createFromTimestamp($duration->getTimestamp()), false) / 60;
```

Comment:
Cache component doesn't load `app()` which is under `Illuminate\Foundation`, nor does it require `illuminate/container`.